### DISCUSSION TITLE
When my bootstrap is upgraded from 5.2.0 to 5.2.1, there will be a problem when the button is active.

### DISCUSSION THREAD

**Discussion by peipei818** [twbs/bootstrap#38378] [Q&A] [answered]
when version is 5.2.0,.btn:active attribute is overridden.
<img width="610" alt="image" src="https://user-images.githubusercontent.com/128784268/228754254-0da992f7-6e84-4480-98c5-ba82c34b7144.png">

But when version is 5.2.1,btn:first-child:active attribute cant overridden. 
<img width="575" alt="image" src="https://user-images.githubusercontent.com/128784268/228754301-98cdb097-24e7-4986-ab90-dcad6a438de0.png">

How should this problem be solved?

**Comment by mdo** [comment, marked as answer]
Upgrade to the latest release. It's been fixed.
